aries_cloudagent/multitenant/admin/tests/test_routes.py

```diff
@@ -206,6 +208,7 @@ async def test_wallet_create_optional_default_fields(self):
         body = {
             "wallet_name": "test",
             "wallet_key": "test",
+            "wallet_source": "",
```

Comment:
```suggestion
            "wallet_source": "wallet_source_test",
```

Reply:
As i understood this test, it was for default values, which should be empty for `wallet_source`?

Reply:
Shouldn't `None` be a default value? :thinking:

Reply:
Yes, probably should be :)

Reply:
Changed to `None`

Reply:
Fixed, set `None` as default value.